<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_product</class>
 <widget class="QDialog" name="add_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm sản phẩm</string>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Thêm sản phẩm</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>251</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lb_tensp">
        <property name="text">
         <string>Tên sản phẩm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_phan_loai">
        <property name="text">
         <string>Phân loại</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_gia_nhap">
        <property name="text">
         <string>Giá nhập</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_gia_xuat">
        <property name="text">
         <string>Giá xuất</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_nha_cung_cap">
        <property name="text">
         <string>Nhà cung cấp</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="le_tensp"/>
      </item>
      <item>
       <widget class="QComboBox" name="cb_danhmuc"/>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_gia_nhap">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_gia_xuat">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_nha_cung_cap"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>168</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pb_save">
      <property name="text">
       <string>Lưu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_cancel">
      <property name="text">
       <string>Hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tv_list_san_pham">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>411</width>
     <height>241</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
